### DISCUSSION TITLE
Getting trouble with recording video from stream

### DISCUSSION THREAD

**Discussion by viet2411** [PyAV-Org/PyAV#931] [1. Help]
**IMPORTANT:** Be sure to replace all template sections {{ like this }} or your issue may be discarded.

## Overview

{{ I want to record video from my drone's stream but I can't encode the video }}

## Expected behavior

{{ The drone sends stream to the computer and the computer encodes the video based on received data }}
![image](https://user-images.githubusercontent.com/39814151/82773168-59ec1980-9e7c-11ea-93fd-235e95b39824.png)

## Actual behavior

{{ Stream using pyav was good, but I can not encode the video}}
![image](https://user-images.githubusercontent.com/39814151/82773128-417bff00-9e7c-11ea-977c-34af20e2045c.png)

Traceback:
```
{{}}
```

## Investigation

{{Fixing my code and looking for help on the internet but no good answers for me }}

## Research

I have done the following:

- [x] Checked the [PyAV documentation](https://pyav.org/docs)

- [x] Searched on [Google](https://www.google.com/search?q=pyav+how+do+I+foo)
- [x] Searched on [Stack Overflow](https://stackoverflow.com/search?q=pyav)
- [x] Looked through [old GitHub issues](https://github.com/PyAV-Org/PyAV/issues?&q=is%3Aissue)
- [ ] Asked on [PyAV Gitter](https://gitter.im/PyAV-Org)
- [x] ... and waited 72 hours for a response.

## Additional context

{{ Add any other context about the problem here. }}

**Comment by marawan31** [comment]
Can you post your encode function? Also can you save a sample of your drone video and post it?
